docs/malformed_files.md

```diff
@@ -0,0 +1,81 @@
+# Dealing with malformed files
+
+
+## Field compatibility
+
+When loading multiple files, the `INFO` and `FORMAT` fields
+from all VCF files are merged together to generate a "representative header",
+which is then used to generate a single BigQuery schema. If the same key is
+defined in multiple files, then its definition must be compatible across files.
+The compatibility rules are as follows:
+
+* Fields are compatible if they have the same `Number` and `Type` fields.
+  Annotation fields (i.e. those specified by `--annotation_fields`) must also
+  have the same `Description`.
+
+* Fields with different `Type` values are compatible in the following cases:
+
+  * `Integer` and `Float` fields are compatible and are converted to `Float`.
+  * You must run the pipeline with `--allow_incompatible_records` to
+    automatically resolve conflicts between incompatible fields (e.g. String
```

Comment:
nit: please put String and Integer in backquotes to match the style of other types.